blocks-common/i-bem/__dom/i-bem__dom.js

```diff
@@ -805,11 +805,23 @@ var DOM = BEM.DOM = BEM.decl('i-bem__dom', {
             ctx = this.findElem(ctx);
         }
 
+        names = names.split(' ');
+
         var _self = this.__self,
-            selector = '.' +
-                names.split(' ').map(function(name) {
-                    return buildClass(_self._name, name, modName, modVal);
-                }).join(',.');
+            keys = names.map(function(name) {
+                return _self.buildClass(name, modName, modVal);
```

Comment:
in bem-bl buildClass is not static: https://github.com/bem/bem-bl/blob/i-bem-dom-caching%40support/2.x/blocks-common/i-bem/__dom/i-bem__dom.js#L55